### ISSUE TITLE
奇怪，所有的容器启动3dpack都是报这个错误

### ISSUE THREAD

**Issue by flycocke** [YanWenKun/ComfyUI-Docker#38]
[Impact Pack] Wildcards loading done.
Traceback (most recent call last):
  File "/root/ComfyUI/nodes.py", line 1906, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/root/ComfyUI/custom_nodes/ComfyUI-3D-Pack/__init__.py", line 44, in <module>
    module = importlib.import_module(f".{nodes_filename}", package=__name__)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/root/ComfyUI/custom_nodes/ComfyUI-3D-Pack/nodes.py", line 52, in <module>
    from .algorithms.dmtet import DMTetMesh
  File "/root/ComfyUI/custom_nodes/ComfyUI-3D-Pack/algorithms/dmtet.py", line 6, in <module>
    from pytorch3d.loss import chamfer_distance, mesh_laplacian_smoothing
  File "/usr/lib64/python3.11/site-packages/pytorch3d/loss/__init__.py", line 8, in <module>
    from .chamfer import chamfer_distance
  File "/usr/lib64/python3.11/site-packages/pytorch3d/loss/chamfer.py", line 11, in <module>
    from pytorch3d.ops.knn import knn_gather, knn_points
  File "/usr/lib64/python3.11/site-packages/pytorch3d/ops/__init__.py", line 7, in <module>
    from .ball_query import ball_query
  File "/usr/lib64/python3.11/site-packages/pytorch3d/ops/ball_query.py", line 10, in <module>
    from pytorch3d import _C
ImportError: /usr/lib64/python3.11/site-packages/pytorch3d/_C.cpython-311-x86_64-linux-gnu.so: undefined symbol: _ZN3c104cuda9SetDeviceEi

Cannot import /root/ComfyUI/custom_nodes/ComfyUI-3D-Pack module for custom nodes: /usr/lib64/python3.11/site-packages/pytorch3d/_C.cpython-311-x86_64-linux-gnu.so: undefined symbol: _ZN3c104cuda9SetDeviceEi
Workspace manager - Openning file hash dict

**Comment by YanWenKun** [comment]
啊，这个估计比较复杂了，我可能得编译一下新版PyTorch3D（之后测试和debug不知道要花多久，也不保证一定能用，因为3D-Pack的兼容性还在PyTorch 2.2.2）。

你可以先试试把PyTorch降级到 2.2.2，这个是一直能用的：
```
pip install --force-reinstall \
    xformers==0.0.25.post1 torchvision torchaudio \
    --index-url https://download.pytorch.org/whl/cu121 \
    --extra-index-url https://pypi.org/simple
```

**Comment by flycocke** [comment]
解决了，谢谢，不过3dpack又更新了，LinuxWheels 需要重新编译了

**Comment by YanWenKun** [comment]
Fixed in 2e76749

@flycocke 刚基于 PyTorch 2.3 编译了新版，更新镜像即可。

@Corshi I have compiled new version of wheels for 3D-Pack. Just update to the latest Docker image, it should work.

**Comment by YanWenKun** [comment]
搞定了，用新版镜像就完事了嗷

Done Update!
